src/bot.ts

```diff
@@ -257,25 +257,20 @@ export class DiscordBot {
     }
     log.info(`Got redact request for ${event.redacts}`);
     log.verbose(`Event:`, event);
+    
     const storeEvent = await this.store.Get(DbEvent, {matrix_id: event.redacts + ";" + event.room_id});
+    
     if (!storeEvent.Result) {
       log.warn(`Could not redact because the event was not in the store.`);
       return;
     }
     log.info(`Redact event matched ${storeEvent.ResultCount} entries`);
     while (storeEvent.Next()) {
       log.info(`Deleting discord msg ${storeEvent.DiscordId}`);
-      if (!this.bot.guilds.has(storeEvent.GuildId)) {
-        log.warn(`Could not redact because the guild could not be found.`);
-        return;
-      }
-      if (!this.bot.guilds.get(storeEvent.GuildId).channels.has(storeEvent.ChannelId)) {
-        log.warn(`Could not redact because the guild could not be found.`);
-        return;
-      }
-      const channel = <Discord.TextChannel> this.bot.guilds.get(storeEvent.GuildId)
-                      .channels.get(storeEvent.ChannelId);
-      const msg = await channel.fetchMessage(storeEvent.DiscordId);
+      const result = await this.LookupRoom(storeEvent.GuildId, storeEvent.ChannelId, event.sender);
```

Comment:
What happens is this fails, should we report it?

Reply:
That already catches and falls back to the normal bot within that function, so i don't think it is needed